Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -2393,6 +2518,30 @@ def init_globals(command_name: str = ''):
                      ' to find the channel.'
     CHANNEL_NOT_FOUND_ERROR_MSG = error_str
 
+    # Handle Long-Running Specific Globals
+    if command_name == 'long-running-execution':
+        # Start event loop for long running
+        loop = asyncio.get_event_loop()
+        if not loop._default_executor:  # type: ignore[attr-defined]
+            demisto.info(f'setting _default_executor on loop: {loop} id: {id(loop)}')
+            loop.set_default_executor(concurrent.futures.ThreadPoolExecutor(max_workers=4))
+
+        # Bot identification
+        integration_context = get_integration_context(SYNC_CONTEXT)
+        if integration_context.get('bot_user_id'):
+            BOT_ID = integration_context['bot_user_id']
+            if BOT_ID == 'null' or BOT_ID is None:
+                # In some cases the bot_id can be stored as a string 'null', this handles this edge case.
+                BOT_ID = get_bot_id()
+                set_to_integration_context_with_retries({'bot_user_id': BOT_ID}, OBJECTS_TO_KEYS, SYNC_CONTEXT)
+        else:
+            BOT_ID = get_bot_id()
+            set_to_integration_context_with_retries({'bot_user_id': BOT_ID}, OBJECTS_TO_KEYS, SYNC_CONTEXT)
+
+        # Pull initial Cached context and set the Expiry
+        CACHE_EXPIRY = int(datetime.now(timezone.utc).timestamp() * 1000) + 300
```

Comment:
this calculation should happen only in one place, please wrap it in a function and use it here and in `fetch_context`.